js/index.js

```diff
@@ -0,0 +1,49 @@
+
+$('#buttons-container').on('click', 'span', printToScreen);
+$('#equals').on('click', calculate);
+$('#clear').on('click', clearScreen);
+
+let entry = [];
+
+function printToScreen(event) {
+  const buttonPushed = $(event.target).text();
```

Comment:
Isolating the information that you would like to work with is a really good pattern.